src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -298,9 +298,19 @@ export default class StandardRouteModel extends BaseModel {
      * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
      */
     getAllFixNamesInUse() {
+        // if data has been incorrectly defined in the airport.json file, we throw and provide a useful message
+        // this should never throw in production, but should be useful for developers as they create and update airport files
+        if (typeof this.draw[0] === 'undefined' || typeof this.draw[0] === 'string') {
```

Comment:
why not a more strict `if (!_isArray(this.draw[0]) {`?

Reply:
because screaming babies.  I should have done it that way, you're right.

Reply:
done